native_client/deepspeech.h

```diff
@@ -15,6 +15,19 @@ struct ModelState;
 
 struct StreamingState;
 
+// Stores each individual character, along with its timing information
+struct MetadataItem {
+  char* character;
```

Comment:
I assume as this is to handle non-English scripts that this can point to a multi-byte character.